llama_hub/asana/base.py

```diff
@@ -19,39 +16,61 @@ def __init__(self, asana_token: str) -> None:
 
         self.client = asana.Client.access_token(asana_token)
 
-    def load_data(self, workspace_id: str) -> List[Document]:
+    def load_data(self, id: str, id_type: str) -> List[Document]:
```

Comment:
curious: why not just have separate `workspace_id` and `project_id`. It's a lot more explicit, and more backward compatible.